### PR TITLE
[Ubuntu] Add AzModule 7.3.2

### PR DESCRIPTION
**Pull request by zoran-bjelica**
# Description
Add AzModule version 7.3.2

#### Related issue:
https://github.com/actions/virtual-environments-internal/issues/3530

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/toolsets/toolset-1804.json [hunk 1]
```diff
@@ -120,7 +120,8 @@
             "name": "az",
             "url" : "https://raw.githubusercontent.com/Azure/az-ps-module-versions/main/versions-manifest.json",
             "versions": [
-                "7.1.0"
+                "7.1.0",
```
Comment (al-cheb) [inline images/linux/toolsets/toolset-1804.json:123]: We should replace Az 7.1.0 -> 7.3.2.

**Comment by miketimofeev** [comment]
@zoran-bjelica We need to replace 7.1.0, not just adding a new one

File: images/linux/toolsets/toolset-1804.json (hunk 1 above)
Comment (zoran-bjelica) [reply images/linux/toolsets/toolset-1804.json]: I have changed version 7.1.0 to 7.3.2

**Review by miketimofeev — APPROVED** [review]

**Review by al-cheb — DISMISSED** [review]

**Comment by mspaansen** [comment]
🙌

Op vr 15 apr. 2022 11:09 schreef Mikhail Timofeev ***@***.***

**Review by vsafonkin — APPROVED** [review]

**Comment by zoran-bjelica** [comment]
Don't approve this, as I have made rebase of my branch and it pulled a lot of not related changes.

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**